pyspectrafuse/cluster_parquet_combine/cluster_res_handler.py

```
@@ -0,0 +1,83 @@
+        charge_cluster_res_groups = {}
+        if charge_groups:
+            print(f"charge_groups: {charge_groups}")
+            for charge, charge_tsv_lst in charge_groups.items():
+                charge_df_lst = []
+                for charge_tsv in charge_tsv_lst:
+                    df = self.read_cluster_tsv(charge_tsv)
+                    sample_info = str(Path(*charge_tsv.parts[-4:-1])).replace("\\",
+                                                                              "/")  # ('Homo sapiens', 'Q Exactive', 'charge2')
+                    # 对于一个聚类结果文件，补上其[物种/仪器/电荷信息], 最终同个电荷的只对应一个parquet,parquet按照电荷切分
+                    df.loc[:, "mgf_path"] = df.loc[:, "mgf_path"].apply(
+                        lambda x: sample_info + "/mgf files/" + x)
```

Ruff

Function definition does not bind loop variable `sample_info`

(B023)